### ISSUE TITLE
[Feature Request] Create an official Helm Chart

### ISSUE THREAD

**Issue by awhitepl** [zerotier/ZeroTierOne#1691]
I think ZT needs an official Helm Chart for installing into Kubernetes clusters. 

It should ideally be installed as a Deployment and be documented on how where to set how many replicasets to run with and launch with a read-only root filesystem with writable files only allowed in /tmp. 

I would suggest using Dockerfile.alpine as a base image but simplify this further and run it as a [distroless container. ](https://github.com/GoogleContainerTools/distroless) to maximize the final container image.

**Comment by someara** [comment]
Good idea! I'm working on our packaging pipeline right now... I can add this once its ready
